docs/content/middlewares/forwardauth.md

```diff
@@ -217,6 +217,57 @@ http:
           - "X-Secret"
 ```
 
+### `authResponseHeadersRegex`
+
+The `authResponseHeadersRegex` option is the regex to match the headers that should be copied from the authentication server to the request. All incoming request's headers matching this regex are deleted from the request before any copy happens. 
```

Comment:
I think it's better to be specific on how the matching behave:

```suggestion
The `authResponseHeadersRegex` option is the regex to match the headers that should be copied from the authentication server to the request. All incoming request's headers matching this regex are deleted from the request before any copy happens.
It allows partial matching of the regular expression in the `accessControlAllowOriginList`.
You should use start of string (`^`) and end of string (`$`) anchors to ensure a full match against the header key.
```

Reply:
Sure, commited 👍